### DISCUSSION TITLE
[FEATURE] Graph (Upvote if want)

### DISCUSSION THREAD

**Discussion by bitnom** [valkey-io/valkey#120] [General]
we love graphs

**Comment by daniel-house** [comment]
Can you be a little more specific, please?

**Reply by madolson** [reply]
Can't you see, they love graphs!

**Reply by HeCorr** [reply]
I think it's safe to assume it means a re-implementation/fork of https://github.com/RedisGraph/RedisGraph

**Reply by virtualirfan** [reply]
IANAL. If I'm not mistaken, even if you have to go all the way back to beginning of the RedisGraph [repo](https://github.com/RedisGraph/RedisGraph), you can only find Apache License, Version 2.0 with Commons Clause Restriction (which means you cannot "sell") switching quickly thereafter to AGPL terms, followed eventually by the "your-choice" between SSPLv1 and RSALv2. For most organizations, SSPLv1 will likely be a non-starter.

Again, IANAL but if the valkey community wants to start a Valkey Graph product, it'll probably need to be a clean room new implementation. I love new code smell, but not a trivial undertaking.
